.travis.yml

```diff
@@ -2,3 +2,6 @@ language: erlang
 sudo: false
 otp_release:
   - 20.2
```

Comment:
We need more OTP versions here.

@tsloughter What is rebar3's minimum supported OTP versions?

Reply:
@ericmj how about mirroring Hex's travis, so 20.2, 19.3, 18.3, and maybe 17.5?

Looks like proper plugin doesn't work on 17.5 (https://travis-ci.org/hexpm/hex_erl/jobs/348937049), I'll investigate. If it's gonna be tricky I think it's OK to skip 17.5 here - hex_erl will still be tested on 17.5 anytime we re-vendor it on Hex (e.g. before each hex_erl release)